examples/plotting/plotting_a_diffraction_pattern.py

```diff
@@ -0,0 +1,43 @@
+"""
+Plotting a Diffraction Pattern
+==============================
+
+This is sometimes not as straightforward as it seems because you might have a zero
+beam that is too bright and regions of the diffraction pattern that are too dark.
+"""
+
+from pyxem.data import fe_multi_phase_grains
+
+mulit_phase = fe_multi_phase_grains()
+
+# %%
+# We can plot easily using the `plot` method. This will show the diffraction pattern
+# but the plot is static and not interactive. Additionally, the zero beam is too bright
+# and the high k values are too dark.
+
+mulit_phase.plot()
+
+# %%
+# Plotting the diffraction pattern with a logarithmic scale can help to see the high k values
+# But because most of the values are zero, the contrast is not great and is too stretched.
+
+mulit_phase.plot(norm="log")
+
+# %%
+# You can also add a small value to the data to avoid the log(0) problem.
+# This doesn't change the underlying data, only creates a new signal for plotting!
+
+(mulit_phase + 1).plot(norm="log")
```

Comment:
How does it compare with `norm="symlog"`? If it works well, would it better to use symlog to avoid creating a new signal?

Reply:
@ericpre That is an excellent idea.  I kind of forget that symlog is an option.  That works much better :)